docs/user-guide/doc-odm-user-guide/import-samples-spreadsheet.md

```diff
@@ -95,43 +121,41 @@ In the subsequent dialog window, several parameters need to be defined:
 
-In the second step, you’ll need to choose the file you wish to upload. There are two options:
+### **Linking Data**
 
-- **Local Computer**: Select the file located on your computer. Please note: this option is only available if your ODM version is coupled with AWS S3 storage (provided by default). If this option isn’t functioning properly, please get in touch with Genestack’s customer care team.
-- **External Link**: Upload the file by supplying a link to the file’s location. Keep in mind that while this option is intended to support any external location, in its BETA version, it only supports links from AWS S3 storage. If you encounter any issues with this feature, please reach out to Genestack’s customer care team.
+* **Default Linking**: By default, the data is linked with the Samples file using the **Sample Source ID** column. To ensure proper linking, make sure your file includes a column called **Sample Source ID** with the same IDs used in the Sample Metadata table uploaded previously (see section "Import Samples Metadata").
+* **Custom Linking**: Alternatively you can select a different column to link the **experimental** data, such as **Sample Name**, **Date**, etc. This provides flexibility in how data is associated, but it is recommended to include the **SampleSourceID** column for consistent referencing and linking samples metadata files with additional data types like libraries and preparations. Read the [Supported format files](https://docs.google.com/document/d/1uF3g11QO3PfyhsSY7u0CPozn0LPRsoblnI2djZydvIs/edit?usp=sharing) section for more information.
 
-![image](doc-odm-user-guide/images/import-data-from-file.png)
+![59LinkData.png](doc-odm-user-guide/images/59LinkData.png)
+<figcaption>Select an experimental data file. The data must include a column to be linked to the sample metadata file (typically the <strong>Sample Source ID</strong>)</figcaption>
 
-Once the data is chosen, the upload will begin. The duration can vary from a few seconds to more, depending on the size of the original file.
+The selected files will be scanned to find an appropriate link (typically the **Sample Source ID** column) and the uploading will automatically begin.
 
-After the upload completes, the corresponding data will be visible in the ‘Data’ tab. All files are grouped by the chosen ‘Data Class’ parameter on the left panel, which includes attached files as well.
+![60Imported.png](doc-odm-user-guide/images/60Imported.png)
+<figcaption>The selected files will be scanned, and if the format is accepted and the columns contain the reference names to be linked, the files will be indexed and the experimental data will be searchable</figcaption>
 
 After uploading, you can populate the corresponding file metadata, including the necessary details. Please note that each uploaded data file has five mandatory read-only fields that do not belong to your template:
```

Comment:
Could you please remove this part: "that do not belong to your template", because these attributes are from template? since we work with this file anyway